packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/widgets.jsx

```diff
@@ -94,8 +119,25 @@ export const MultiplierOptionsWidget = observer(() => {
 });
 
 export const AccumulatorOptionsWidget = observer(() => {
-    const { growth_rate } = useTraderStore();
+    const { growth_rate, has_open_accu_contract, tick_size_barrier } = useTraderStore();
     const displayed_trade_param = `${getGrowthRatePercentage(growth_rate)}%`;
     const modal_title = localize('Growth rate');
-    return <RadioGroupOptionsWidget displayed_trade_param={displayed_trade_param} modal_title={modal_title} />;
+    const extra_tooltip_message = (
+        <Localize
+            i18n_default_text='Your stake will grow at {{growth_rate}}% per tick as long as the current spot price remains within ±{{tick_size_barrier}} from the previous spot price.'
+            values={{
+                growth_rate: getGrowthRatePercentage(growth_rate),
+                tick_size_barrier: getTickSizeBarrierPercentage(tick_size_barrier),
+            }}
+        />
+    );
+    return (
+        <RadioGroupOptionsWidget
+            displayed_trade_param={displayed_trade_param}
+            is_disabled={has_open_accu_contract}
+            modal_title={modal_title}
+            should_show_extra_tooltip
```

Comment:
```suggestion

```
@kate-deriv nit: we can get rid of this and rely on extra_tooltip_message. :) If !!extra_tooltip_message, we will do whatever you do now when should_show_extra_tooltip is true